feat(content): Adjust Firelight description (#8153)
endless-sky/endless-sky: data/korath/korath weapons.txt

		"missile strength" 60
		submunition "Firelight Turning"
		"stream"

> 	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels. This newer, more compact version is called the Firelight missile."

Reviewer: That blurb is just is just redundant on several levels.  Here:

```suggestion
	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to create the Firelight by further scaling down the reactor element to fit in a missile casing for use on and against smaller vessels."
```

Author: Yeah, definitely reads better.

> 	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels. These newer, more compact Firestorms, are called Firelight missiles."

Reviewer 2: ```suggestion
	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels. This newer, more compact version is called the Firelight missiles."
```
